network/peers/connections/handshaker.go

```diff
@@ -187,22 +206,27 @@ func (h *handshaker) Handshake(logger *zap.Logger, conn libp2pnetwork.Conn) erro
 	if err := h.preHandshake(conn); err != nil {
 		return errors.Wrap(err, "could not perform pre-handshake")
 	}
-	sni, err := h.nodeInfoFromStream(logger, conn)
+
+	var ani records.AnyNodeInfo
+
+	ani, err = h.nodeInfoFromStream(logger, conn)
```

Comment:
do we need `var` initiation here?
We can just `ani, err := h.nodeInfoFromStream(logger, conn)`

Reply:
you are right its not a must